### ISSUE TITLE
fresh nodecg-cli on node.js v.0.12 gives : No such file or directory

### ISSUE THREAD

**Issue by zyxep** [nodecg/nodecg-cli#10]
Hi.

A fresh install of nodecg-cli

```
$ sudo npm install -g nodecg-cli
[sudo] password for zyxep:
/usr/local/bin/nodecg -> /usr/local/lib/node_modules/nodecg-cli/bin/nodecg
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/npmconf requires semver@'2' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/semver,
npm WARN unmet dependency which is version 4.2.0
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/npmconf requires uid-number@'0.0.5' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/uid-number,
npm WARN unmet dependency which is version 0.0.6
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/read-package-json/node_modules/normalize-package-data requires semver@'2 || 3' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/semver,
npm WARN unmet dependency which is version 4.2.0
nodecg-cli@2.1.0 /usr/local/lib/node_modules/nodecg-cli
├── rimraf@2.2.8
├── commander@2.6.0
├── chalk@0.5.1 (escape-string-regexp@1.0.2, ansi-styles@1.1.0, supports-color@0.2.0, has-ansi@0.1.0, strip-ansi@0.3.0)
├── semver@4.3.0
├── npm-package-arg@3.1.0 (hosted-git-info@1.5.3)
├── fs.extra@1.3.2 (mkdirp@0.3.5, walk@2.3.9, fs-extra@0.6.4)
├── request@2.53.0 (caseless@0.9.0, json-stringify-safe@5.0.0, forever-agent@0.5.2, aws-sign2@0.5.0, stringstream@0.0.4, oauth-sign@0.6.0, tunnel-agent@0.4.0, isstream@0.1.1, node-uuid@1.4.2, qs@2.3.3, combined-stream@0.0.7, form-data@0.2.0, mime-types@2.0.9, http-signature@0.10.1, bl@0.9.4, tough-cookie@0.12.1, hawk@2.3.1)
└── inquirer@0.8.0 (ansi-regex@1.1.0, figures@1.3.5, mute-stream@0.0.4, through@2.3.6, readline2@0.1.1, lodash@2.4.1, cli-color@0.3.2, rx@2.4.0)

```

Gives me following

```
$ nodecg
: No such file or directory
```

**Comment by MattMcNam** [comment]
Should be fixed in the 2.1.1 release just published. Rerun the install command to update to it and reopen if it's still broken!

**Comment by zyxep** [comment]
@MattMcNam it works :)
